py3status/modules/yandexdisk_status.py

```diff
@@ -0,0 +1,71 @@
+# -*- coding: utf-8 -*-
+"""
+Display Yandex.Disk status.
+
+Configuration parameters:
+    cache_timeout: how often we refresh this module in seconds (10s default)
+    format: prefix text for the Yandex.Disk status
+
+Valid status values include:
+    - Yandex.Disk isn't running!
+    - Error: daemon not started
+    - ...
+
+Requires:
+    Yandex.Disk CLI: command line tool (link: https://disk.yandex.com/)
+
+@author Vladimir Potapev (github:vpotapev)
+@license BSD
+"""
+
+import shlex
+import subprocess
+from time import time
+
+
+class Py3status:
+    """
+    """
+    # available configuration parameters
+    cache_timeout = 10
+    format = 'Yandex.Disk: {}'
+
+    def yadisk(self, i3s_output_list, i3s_config):
+        response = {'cached_until': time() + self.cache_timeout}
+
+        raw_lines = ''
+        try:
+            raw_lines = subprocess.check_output(shlex.split('yandex-disk status'))
+        except subprocess.CalledProcessError as e:
+            if e.returncode == 1:
```

Comment:
@pydsigner 
If `e.returncode != 1`, the raw_lines variable will be empty string (it is defined at line 36). So it seems it should work correctly. But maybe I've missed something, and you have meant another problem?